### DISCUSSION TITLE
Error on upload ADIF

### DISCUSSION THREAD

**Discussion by 112bzo** [magicbug/Cloudlog#1224] [General]
Dear fellow amateurs,

When I upload my ADIF file in Cloudlog I see the following error message. How can I solve this or what is the solution
![upload-cloudlog](https://user-images.githubusercontent.com/43230470/136845325-8482841b-b4ca-430e-b257-4547fa5f5f3f.png)
?

**Comment by magicbug** [comment]
Without seeing the ADIF its impossible to say

**Comment by 112bzo** [comment]
cant load my ADIF file here?

**Comment by magicbug** [comment]
If you rename it .txt and drag into the text area it will upload it

**Comment by 112bzo** [comment]
Oke here a sample:

QRZLogbook download for pd9dp
    Date: Mon Oct  4 15:33:48 2021
    Bookid: 274646
    Records: 3391
    <ADIF_VER:5>3.1.1
    <PROGRAMID:10>QRZLogbook
    <PROGRAMVERSION:3>2.0
    <eoh>
<app_qrzlog_logid:9>546933520
<app_qrzlog_status:1>N
<band:4>70cm
<band_rx:4>70cm
<call:6>PI4RCG
<comment:8>Balonvos
<cont:2>EU
<country:11>Netherlands
<cqz:2>14
<distance:4>5749
<dxcc:3>263
<email:16>PE1IYR@AMSAT.ORG
<eqsl_qsl_rcvd:1>N
<eqsl_qsl_sent:1>N
<freq:7>430.125
<freq_rx:7>430.125
<gridsquare:6>JO22of
<ituz:2>27
<lat:11>S000 00.000
<lon:11>W000 00.000
<lotw_qsl_rcvd:1>N
<lotw_qsl_sent:1>Y
<lotw_qslsdate:8>20210704
<mode:2>FM
<my_city:6>Deurne
<my_country:11>Netherlands
<my_cq_zone:1>1
<my_gridsquare:8>JO21VL61
<my_itu_zone:2>27
<my_lat:11>N051 27.998
<my_lon:11>E005 48.376
<my_name:14>Theo Vermeulen
<name:17>VERON AFD. T GOOI
<qrzcom_qso_upload_date:8>20200920
<qrzcom_qso_upload_status:1>Y
<qsl_rcvd:1>N
<qsl_sent:1>N
<qsl_via:5>PA0MW
<qso_date:8>20200920
<qso_date_off:8>20200920
<qth:16>1213AE HILVERSUM
<rst_sent:3>599
<station_callsign:5>PD9DP
<time_off:4>1335
<time_on:4>1335
<eor>

<app_qrzlog_logid:9>549650516
<app_qrzlog_qsldate:8>20210704
<app_qrzlog_status:1>C
<band:3>40m
<band_rx:3>40m

**Comment by magicbug** [comment]
I've a feeling this wiki item addresses what might be happening https://github.com/magicbug/Cloudlog/wiki/ADIF-file-can%27t-be-imported

**Comment by AndreasK79** [comment]
You could also try to validate your ADIF-file here: https://www.rickmurphy.net/adifvalidator.html

**Comment by AndreasK79** [comment]
I found another one here: http://hamclubs.info/adif-validator/
